src/cli/vector_search.rs

```diff
@@ -34,11 +34,6 @@ impl Command for VectorSearch {
     fn signature(&self) -> Signature {
         Signature::build("vector search")
             .required("index", SyntaxShape::String, "the index name")
-            .required(
```

Comment:
I suspect that this should still be an input field. See `doc get` where it supports piping but is also an optional field.